Installing on Raspberry Zero W

Hi,

I'm receiving the following error when trying to install the library on Raspberry Zero W with Node.js `v16.17.0`.

```
npm WARN deprecated node-pre-gyp@0.15.0: Please upgrade to @mapbox/node-pre-gyp: the non-scoped node-pre-gyp package is deprecated and only the @mapbox scoped package will recieve updates in the future
npm ERR! code 1
npm ERR! path /home/pi/example/node_modules/aws-crt
npm ERR! command failed
npm ERR! command sh /tmp/install-4ef19b8a.sh
npm ERR! /home/pi/example/node_modules/aws-crt/scripts/build.js:154
npm ERR!     if (fs.existsSync(nativeSourceDir)) {
npm ERR!                       ^
npm ERR!
npm ERR! ReferenceError: nativeSourceDir is not defined
npm ERR!     at buildFromRemoteSource (/home/pi/example/node_modules/aws-crt/scripts/build.js:154:23)
npm ERR!     at main (/home/pi/example/node_modules/aws-crt/scripts/build.js:193:13)
npm ERR!     at Object.<anonymous> (/home/pi/example/node_modules/aws-crt/scripts/build.js:205:3)
npm ERR!     at Module._compile (node:internal/modules/cjs/loader:1126:14)
npm ERR!     at Object.Module._extensions..js (node:internal/modules/cjs/loader:1180:10)
npm ERR!     at Module.load (node:internal/modules/cjs/loader:1004:32)
npm ERR!     at Function.Module._load (node:internal/modules/cjs/loader:839:12)
npm ERR!     at Module.require (node:internal/modules/cjs/loader:1028:19)
npm ERR!     at require (node:internal/modules/cjs/helpers:102:18)
npm ERR!     at Object.<anonymous> (/home/pi/example/node_modules/aws-crt/scripts/install.js:20:1)
npm ERR!     at Module._compile (node:internal/modules/cjs/loader:1126:14)
npm ERR!     at Object.Module._extensions..js (node:internal/modules/cjs/loader:1180:10)
npm ERR!     at Module.load (node:internal/modules/cjs/loader:1004:32)
npm ERR!     at Function.Module._load (node:internal/modules/cjs/loader:839:12)
npm ERR!     at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)
npm ERR!     at node:internal/main/run_main_module:17:47

```

Do you happen to have an idea why this is happening and how to fix this?

`package.json` contains following entries:

```
    "aws-crt": "^1.14.3",
    "aws-iot-device-sdk-v2": "^1.8.7",
```

The code works just fine on a macOS.

--- a/scripts/build.js
+++ b/scripts/build.js
@@ -12,6 +12,8 @@ const cmake = require("cmake-js");
 const axios = require("axios");
 const tar = require("tar");
 
+const nativeSourceDir = "crt/";
+
 function rmRecursive(rmPath) {
     let rmBasePath = path.basename(rmPath);
     if (rmBasePath == "." || rmBasePath == "..") {
@@ -185,7 +187,6 @@ function checkDoDownload() {
     // Makes sure the work directory is what we need
     const workDir = path.join(__dirname, "../")
     process.chdir(workDir);
-    const nativeSourceDir = "crt/"
 
     if (checkDoDownload()) {
         const tmpPath = path.join(__dirname, `temp${crypto.randomBytes(16).toString("hex")}/`);

